docs/CODEOWNERS

```diff
@@ -0,0 +1,351 @@
+# CODEOWNERS
```

Comment:
Weird that github doesn't detect that this file is a moved version of the /CODEOWNERS (with some text removed)...